packages/core/jkl/_typography.scss

```diff
@@ -241,15 +241,19 @@ $text-styles: (
 
         @if string.length($variant) == 0 {
             @each $property, $value in map.get($variants, "small-screen") {
-                #{$property}: $value;
+                & {
+                    #{$property}: $value;
+                }
```

Comment:
Dette vil så vidt jeg kan forstå lage en selector per egenskap, så hvis vi klarer å unngå å bruke `$ { /* ... */ }` hadde det vært nice. Funker det ikke å flytte `jkl.text-style()` til etter de vanlige reglene, sånn som med de andre mixin-ene?

Reply:
Evt. kan man kanskje sette `& {` rundt hele `@if`-løkken?

Reply:
Fant ut av det! Se forslag under 😊 Vi bør sikkert skrive en liten oppskrift for fiks ute i pakkene. Kanskje også legge til info i Sass-doc på mixin-ene som legger til nøstede selektorer så det blir litt lettere i fremtiden.

Reply:
Nice!